homeassistant/core.py

```diff
@@ -2742,6 +2751,7 @@ def __init__(self, hass: HomeAssistant, config_dir: str) -> None:
         self.elevation: int = 0
         """Elevation (always in meters regardless of the unit system)."""
 
+        self.debug: bool = False
```

Comment:
Can we default this to the debug value in [`RuntimeConfig`](https://github.com/home-assistant/core/blob/dev/homeassistant/runner.py#L55)

Reply:
Yes

18e88c20d9d53f0ce45e12b667d66e7df4c478a6